tests/microos/install_image.pm

```diff
@@ -64,23 +71,28 @@ sub run {
     assert_script_run("btrfs property set /mnt ro true");
     assert_script_run("umount /mnt");
 
-    # Setup ignition parition on the end of the same disk and resize root partition to use all the space
-    # script_output recommended in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/20253/files#r1776549682
-    script_output("printf \"fix\n\" | parted ---pretend-input-tty ${device} print");
-    assert_script_run("parted ${device} --script mkpart primary ext4 98% 100%");
-    assert_script_run("parted ${device} --script print");
-    assert_script_run("mkfs.ext4 -F ${ignition_partition}");
-    assert_script_run("e2label ${ignition_partition} ignition");
-    assert_script_run("mount ${ignition_partition} /mnt/");
-    assert_script_run("mkdir /mnt/ignition");
-    assert_script_run("curl -v -o /mnt/ignition/config.ign " . data_url("microos/ignition/config.ign"));
-    assert_script_run('umount /mnt');
-
-    # Resize root filesystem to maximum size to use all space up to partition with ignition
-    assert_script_run("parted ${device} --script resize ${root_partition_id} 98%");
-    assert_script_run("mount ${root_partition} /mnt");
-    assert_script_run("btrfs filesystem resize max /mnt");
-    assert_script_run("umount /mnt");
+    my $main_partition_size = '100%';
+    unless (check_var('FIRST_BOOT_CONFIG', 'wizard')) {
+        # Setup ignition parition on the end of the same disk and resize root partition to use all the space
+        # script_output recommended in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/20253/files#r1776549682
+        $main_partition_size = '98%';
```

Comment:
why is it defined as variable now? and why is it called `main_partition_size`? isn't it `root`?